lumicks/pylake/force_calibration/power_spectrum_calibration.py

```diff
@@ -285,10 +291,13 @@ def fit_power_spectrum(power_spectrum, model, settings=CalibrationSettings()):
                 "Chi squared per degree of freedom", chi_squared_per_deg, "-"
             ),
             "backing (%)": CalibrationParameter("Statistical backing", backing, "%"),
+        },
+        params={
+            **model.calibration_parameters(),
+            "Model": CalibrationParameter("Calibration model", model.__name__(), "-"),
```

Comment:
It would be nice to drop the model from `results.params` since already available as `results.model`. It's also a string in a `dict` where everything else is a number so it requires special handling for some things.

Reply:
Dropped.